integrations/pom.xml

```diff
@@ -16,27 +16,21 @@
     <jackson.version>2.14.2</jackson.version>
     <enforcer.skip>true</enforcer.skip> <!-- I give up, too much effort to maintain enforcer here, should probably try clear out all enforcer only deps and see if it works -->
     <netty.version>4.1.81.Final</netty.version>
-    <jenkins.version>2.387.1</jenkins.version>
```

Comment:
Using `jenkins.version` from the multi-module project root. Too confusing to have multiple `jenkins.version` values in different modules: if a newer version is needed, just bump it in the multi-module project root and be done with it.

Reply:
I guess with the tests now (or almost) being run in bom we can remove https://github.com/jenkins-infra/release/blob/c82d88610c99d0983a556c8b4d09063bb93d978a/.github/ISSUE_TEMPLATE/1-lts-release-checklist.md#L38

Reply:
Ya that sounds like a good idea. Which means we can most likely also cleanup this:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.github/workflows/updatecli-action.yaml
https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/updatecli